README.md

```diff
@@ -131,6 +136,25 @@ Then follow the steps to [run Atoti UI](https://activeviam.com/activeui/document
 
 Finally open your dashboards, widgets, filters, ... Make sure they work as expected.
 
+## Migrate an Atoti jupyter notebook
+
+To migrate a jupyter notebook, two files are necessary:
+
+- _notebook.ipynb_, the notebook to migrate. It is really important that the notebook has the `.ipynb` extension.
+- _servers.json_, the JSON file containing the server information.
+
+To get the servers information, follow the [download the servers information section](#download-the-servers-information) instructions.
+
+Now that your migration folder contains all the required file, you can run the following CLI:
+
+```
+npx -- activeui-migration migrate -i notebook.ipynb -o migrated-notebook.ipynb -s servers.json --from-version 0.7 --to-version 0.8
+```
+
+This command generates a file named `migrated-notebook.json` in the same folder. It contains the migrated notebook with all its migrated widgets, ready to be used with the Atoti version you migrated to. You can see the [run the CLI section](#run-the-cli) to get more details about the command line and its options.
+
+:warning: Calculated measures are not handled by the migration. In that case, you should update manually the widgets containing the calculated measures.
```

Comment:
I'd like to be a bit more specific and avoid generic words like _"handle"_ or _"update"_, in order to make the sentence clearer. In this case, we are talking about removing saved calculated measures from widgets queries.

```suggestion
:warning: Saved calculated measures are not removed from the queries of the widgets in your notebook. If you migrate your Content Server on top of your notebook, then you have to remove these saved calculated measures from the queries in your notebook manually (if any).
```